src/utils/file.js

```diff
@@ -34,3 +34,17 @@ export function getTypedFolderId(id: string): string {
 export function isBoxNote(file: BoxItem): boolean {
     return file.extension === FILE_EXTENSION_BOX_NOTE;
 }
+
+/**
+ * Returns the extension from the file name
+ * @param {string} filename a box file
+ * @return {string} typed id for file
+ */
+export function getFileExtension(filename: string): string | null {
+    if (typeof filename !== 'string') return null;
+
+    // case insensitive regex to extract file extension without .
+    const regex = /\.([0-9a-z]+)$/i;
+    const result = regex.exec(filename);
+    return result && result[1] ? result[1] : null;
```

Comment:
This could be simplified to something like `return (result && result[1]) || null;`